iotdb/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -253,13 +253,13 @@
    */
   private int maxLogEntrySize = 4 * 1024 * 1024;
   /**
-   * Is this IoTDB instance a receiver of postback or not.
+   * Is this IoTDB instance a receiver of sync or not.
    */
-  private boolean isPostbackEnable = true;
+  private boolean isSyncEnable = true;
   /**
-   * If this IoTDB instance is a receiver of postback, set the server port.
+   * If this IoTDB instance is a receiver of sync, set the server port.
    */
-  private int postbackServerPort = 5555;
+  private int syncServerPort = 5555;
```

Comment:
do you need to modify the docs?

Reply:
Currently, docs don't have a section to introduce sync util.